backend/Models/Company.php

```diff
@@ -17,6 +17,8 @@ class Company implements JsonSerializable
     protected string $phoneNumber;
     protected string $isPaid;
     protected string $logoFile;
+    protected string $socialsFacebook;
+    protected string $socialsLinkedIn;
```

Comment:
What if there will be `x` more socials? Maybe create Socials class, and some pre-defined names for socials, and play with an array? I don't know business requirements, so this solution may be ok.

Reply:
It's simple change, but might be better in the long run. We don't even need an array, it will automatically be created in json from new class.

Reply:
It now looks nice in ,json file.

![Screenshot from 2021-07-26 13 31 27](https://user-images.githubusercontent.com/49527545/126982424-924bca7b-39d2-4f77-985e-9a94965c2ace.png)

It's in simple object format in PHP, but it's easier to modify and use, and I prefer to keep .csv fields in strict format to limit user error, especially because we want to use corresponding social medias' icons.